worker-message-prioritization-distribution/src/main/java/com/github/workerframework/workermessageprioritization/redistribution/DistributorModule.java

```diff
@@ -224,7 +230,8 @@ Map<String,Object> provideTargetQueueArgs(final MessageDistributorConfig message
     @Override
     protected void configure() {
         bind(MessageDistributorConfig.class).in(Scopes.SINGLETON);
-        bind(TargetQueueSettingsProvider.class).to(K8sTargetQueueSettingsProvider.class);
```

Comment:
Drop the `K8sTargetQueueSettingsProvider` import that is no longer needed following this change.